modules/salary/view/employeereport.php

```diff
@@ -320,23 +329,43 @@ function print_sums($employee_id, $codes, $report) {
     print("</tr>");
 }
 
-print('
-<tr>
-  <td><b>sum fratrukkede l&oslash;nnslipper</b></td>
-');
-$sum_account2 = array();
+$sum_account_prev = array();
+$sum_account_next = array();
 foreach($codes as $c) {
-    $sum = 0;
+    $sum_prev = 0;
     foreach($accountreports as $accountreport) {
         if(intval($accountreport['DifferentYear']) === 0) continue;
-        $sum += $accountreport['amounts'][$c];
+        if($accountreport['JournalYear'] != ($year-1)) continue;
```

Comment:
Instead of two foreach loops have an else branch here and calculate both sum_next and sum_prev in the same foreach loop